src/Illuminate/Database/Schema/Builder.php

```diff
@@ -46,6 +46,14 @@ class Builder
      */
     public static ?int $defaultTimePrecision = 0;
 
+    /**
+     * The default id key type.
+     *
+     * @var string
+     */
+    public static $defaultIdType = 'int';
```

Comment:
Even though, it's done similarly with the default morph column type and the mapping to big integer mostly makes sense, what if, for some reason, someone can not or doesn't want to use the non-big integer type?